co.electriccoin.swiftlint.yml

```diff
@@ -0,0 +1,139 @@
+# This SwiftLint file is based on this great guideline. 
+# https://github.com/raywenderlich/swift-style-guide
+
+excluded:
+  - ${PWD}/Pods
+  - ${PWD}/xctemplates
+
+disabled_rules:
+  - notification_center_detachment
+  - orphaned_doc_comment
+  - todo
```

Comment:
Do we want to disable todos?

I managed to configure a regex for detekt (a similar static analysis tool for Kotlin code), that allows todos if they have an issue number associated with them.  E.g. `TODO [#123]: This doesn't correctly support LTR languages` wouldn't trigger a failure but a plain `TODO: This doesn't correctly support LTR languages` would trigger a failure.

Does SwiftLint support anything similar?

Reply:
I don't know. probably this is possible by adding a custom rule as the ones @adamstener added

Reply:
This would be custom yes but could likely use the same regex

Reply:
probably that's for the best!

Reply:
done https://github.com/zcash/secant-ios-wallet/issues/25